### ISSUE TITLE
Cannot read property 'videoRes' of undefined

### ISSUE THREAD

**Issue by mxmtsk** [ZeroCho/react-filepicker#55]
Hey,
I'm trying to use react-filepicker but I keep getting "Cannot read property 'videoRes' of undefined" and I can't figure out why.

This is my setup:

In my Form Field Component:
```
<ReactFilepicker apikey={filestack} onSuccess={this.changeInput} options={{
                    buttonText: 'Upload picture',
                    buttonClass: 'btn btn-primary',
                    services: ['COMPUTER', 'CONVERT'],
                    conversions: ['crop', 'rotate', 'filter'],
                    mimetype: 'image/*',
                    container: 'modal',
                    imageDim: [450, 450],
                    cropRatio: 1,
                    cropForce: true,
                    multiple: false,
                    location: 'S3',
                    access: 'public',
                    path: `user_avatars/${moment().format('DD-MM-YYYY')}/`,
                  }} />
```

Any clue why this happens? I can't figure it out...

**Comment by ZeroCho** [comment]
Im now on a leave and will come back on Sunday. If you find out the solution, please make a PR or I will find out the solution on sunday

**Comment by ZeroCho** [comment]
@mxmtsk Fixed in 0.10.1. Thank you for telling me this problem.
